input/fsh/modules/participant.fsh

```diff
@@ -94,3 +86,59 @@ Description: "Research oriented patient"
 * extension contains AgeAtLastVitalStatus named age-at-last-vital-status 0..1
 * extension[age-at-last-vital-status] ^short = "Age or date of last vital status"
 
+
+
+//  Person Module
+
+Logical: CdmPerson
+Id: SharedDataModelPerson
+Title: "Shared Data Model for Research Persons"
+Description: "The **Shared data model for Person**"
+* identifier 1..1 string "Unique Person identifier."
+* participant 1..1 reference "The participant we are describing"
+
+Extension: AccessPolicy
+Id: access-policy
+Title: "Access policy"
+Description: "Access Policy Extension"
+* insert SetContext(ResearchSubject)
+* value[x] only Reference
+* valueReference 1..1
+* valueReference only Reference(Consent)
+
+Profile: NcpiPerson
+Parent: Person
+Id: ncpi-person 
+Title: "NCPI Person"
+Description: "Person"
+* ^version = "0.1.0"
+* ^status = #draft
+* id 0..1 
```

Comment:
ID should be 1..1.  (Robert, if you followed an email because you were tagged here, I was looking at the multivalue from the next line in the sheet, so...feel free to ignore this comment! :) )